openwisp_controller/subnet_division/admin.py

```diff
@@ -21,10 +21,23 @@
 
 
 class SubnetDivisionRuleInlineAdmin(
-    MultitenantAdminMixin, TimeReadonlyAdminMixin, admin.StackedInline
+    MultitenantAdminMixin, TimeReadonlyAdminMixin, HelpTextStackedInline
 ):
     model = SubnetDivisionRule
     extra = 0
+    help_text = {
+        'text': _(
+            'Please keep in mind that once the subnet division rule is created '
+            'changing changing "Size", "Number of Subnets" or decreasing '
+            '"Number of IPs" will not be possible.'
+        ),
+        'documentation_url': (
+            'https://github.com/openwisp/openwisp-controller/tree/1.0.x#important-notes'
```

Comment:
please change this to point to the section `Limitations of Subnet Division`, I would also point this link to master, since we want to push this there soon.